app/infrastructure/database/migrations/001_jobs_create.rb

```diff
@@ -6,16 +6,17 @@
   change do
     create_table(:jobs) do
       # Method name # Column
-      primary_key :id
+      primary_key :db_id
 
-      String      :job_id, unique: true
+      Integer     :job_id, unique: true
       String      :job_title
       String      :description
       String      :location
       Float       :min_year_salary
       Float       :max_year_salary
       String      :currency
       String      :url, null: true
+      Bool        :isfull
```

Comment:
Highly recommend following the snake case. `is_full`

Reply:
Issue solved. Please refer to this [commit](https://github.com/KeroKnows/s-killer/commit/6df43afb8140f4ee8e82fc5f9e29205f984a01ce).
Also, you need to re-migrate the database if you would like to run the tests.
Thank you for all the work again!